tests/unit/test-extension-runners/test.extension-runners.js

```diff
@@ -301,6 +301,7 @@ describe('util/extension-runners', () => {
         onSourceChange: sinon.spy(() => {}),
         ignoreFiles: ['path/to/file', 'path/to/file2'],
         reloadExtension: sinon.spy(() => Promise.resolve()),
+        reloadDelay: new Number(1000),
```

Comment:
Why was this added?

`reloadDelay` should be an optional parameter. The `reloadDelay` option should not be set if the default value works for the test.